docs/batch-changes/view-batch-changes.mdx

```diff
@@ -0,0 +1,33 @@
+# Viewing Batch Changes
+
+<p className="subtitle">Learn how to view, search, and filter your Batch Changes.</p>
+
+## Viewing batch changes
+
+You can view a list by clicking the **Batch Changes** icon in the top navigation bar:
```

Comment:
View a list of what? (Just being pedantic here to highlight that it's a little unclear)